examples/firepad.js

```diff
@@ -2495,26 +2495,32 @@ firepad.ACEAdapter = (function() {
   };
 
   ACEAdapter.prototype.operationFromACEChange = function(change) {
-    var action, delta, inverse, operation, restLength, start, text, _ref, _ref1;
+    var action, delete_op, delta, insert_op, restLength, start, text, _ref;
+    if (change.data) {
     delta = change.data;
-    if ((_ref = delta.action) === "insertLines" || _ref === "removeLines") {
-      text = delta.lines.join("\n") + "\n";
-      action = delta.action.replace("Lines", "");
+      if ((_ref = delta.action) === 'insertLines' || _ref === 'removeLines') {
+        text = delta.lines.join('\n') + '\n';
+        action = delta.action.replace('Lines', '');
     } else {
       text = delta.text.replace(this.aceDoc.getNewLineCharacter(), '\n');
-      action = delta.action.replace("Text", "");
+        action = delta.action.replace('Text', '');
     }
     start = this.indexFromPos(delta.range.start);
+    } else {
+      text = change.lines.join('\n');
+      start = this.indexFromPos(change.start);
+    }
     restLength = this.lastDocLines.join('\n').length - start;
-    if (action === "remove") {
+    if (change.action === 'remove') {
       restLength -= text.length;
     }
-    operation = new firepad.TextOperation().retain(start).insert(text).retain(restLength);
-    inverse = new firepad.TextOperation().retain(start)["delete"](text).retain(restLength);
-    if (action === 'remove') {
-      _ref1 = [inverse, operation], operation = _ref1[0], inverse = _ref1[1];
+    insert_op = new firepad.TextOperation().retain(start).insert(text).retain(restLength);
+    delete_op = new firepad.TextOperation().retain(start)["delete"](text).retain(restLength);
+    if (change.action === 'remove') {
+      return [delete_op, insert_op];
+    } else {
+      return [insert_op, delete_op];
     }
-    return [operation, inverse];
```

Comment:
yes. I merged it with latest firepad (1.3) so if you diff it to the 1.3 version you'll see only the change. 

In general I think it's important the repository always contain an updated version of the combined firepad.js. One reason for that is that it alows you to fork it and then test with the forked firepad.js or point a jsfiddle to it (as I've done here). 

I chose the one in examples because it's already included in the release so may as well have it updated.

Hope it makes sense